lib/vcap_env.rb

```diff
@@ -0,0 +1,13 @@
+require 'json'
+
+# Parses ENV['VCAP_SERVICES'] or similar JSON and sets ENV variables for user-provided:credentials
+module VcapEnv
+  def self.set_env(vcap_json)
+    vcap = JSON.parse(vcap_json)
+    env = vcap["user-provided"].find { |service| service["name"] == "compliance-viewer-env"  }
```

Comment:
Just to clarify - I'm good with leaving this as a 'convention' for the time being, i.e. leaving `compliance-viewer-env` hardcoded here. We can revisit this setting later.